README.md

```diff
@@ -62,12 +62,76 @@ Use `action-validator -h` to see additional options.
 > repo -- or, heaven forfend, outside the repository entirely.
 
 
+## Pre-commit hook example
+
+Create an executable file in the .git/hooks directory of the target repository:
+`touch .git/hooks/pre-commit && chmod +x .git/hooks/pre-commit` and paste the following example code:
+
+```bash
+#!/bin/bash
+if [ -d ".github/workflows" ]; then
+  if command -v action-validator >/dev/null 2>&1; then
```

Comment:
Don't suppress stderr here; if `command` prints anything to stderr, it'll be important.

Reply:
I think you're responding to a different comment here; this line just needs the `2>&1` removed.